### DISCUSSION TITLE
Our Next Moves

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [collab-community/journey-book#16] [Show and tell]
@KendallDoesCoding 
-- from now , in this section, we will write and plan our next move

-- whatever we are going to do,even a small change or issue, just give a brief about that in this section. 

-- this will help us to understand everything in a structured way :)

**Comment by KendallDoesCoding** [comment]
> @KendallDoesCoding -- from now , in this section, we will write and plan our next move
> 
> -- whatever we are going to do,even a small change or issue, just give a brief about that in this section.
> 
> -- this will help us to understand everything in a structured way :)

Ok cool! I would suggest we take this to wiki, but here's fine too so anyone can hop in, than the wiki that only collabs can hop in!

**Comment by Adarsh-jaiss** [comment]
-- Now I think, we have set up almost every basic stuff needed.
-- Now its time to work on the website :)

**Reply by Adarsh-jaiss** [reply]
but before that let's sumarize everything and make a clear note

@kendalldoescoding

so, we have done with the 
-- workflow bots (Security & Greeting)
-- code of conduct (coc)
-- pr templates
-- issues templates
-- removed GitHub pages for now until website is ready
-- we have selected option b for the website.

**Reply by Adarsh-jaiss** [reply]
Now we are almost done with the basic stuffs

lets's see what we have to do next
-- We have to format the README a little better #2 (Add badges, grammer check, remove extra sections, etc) **(pr made)**
-- you are going to work on HTML and CSS
-- I have approached riti and techstudent11 for javascript

**Reply by Adarsh-jaiss** [reply]
@KendallDoesCoding 
Challenges we are facing :
1. Adding automation to the website .

**Reply by KendallDoesCoding** [reply]
> -- after the website is ready , we need to deploy it via netfly, linktree or via custom domain

How can you deploy the website to Linktree? I was giving a example of Linkfree, the repo EddieHubCommunity has xD... I'll get the domain for ya if you dont have the GitHub Student Plan but I suggest you get it.

**Reply by Adarsh-jaiss** [reply]
-- sorry for that, i actually don't know much about linktree 🙂

i'll get that soon after I enter in my college🙂

**Reply by KendallDoesCoding** [reply]
> -- sorry for that, i actually don't know much about linktree 🙂
> 
> i'll get that soon after I enter in my college🙂

Ok cool, your on break rn?

**Reply by KendallDoesCoding** [reply]
> Now we are almost done with the basic stuffs
> 
> lets's see what we have to do next -- We have to format the README a little better #2 (Add badges, grammer check, remove extra sections, etc) -- you are going to work on HTML and CSS -- I have approached riti for javascript -- after the website is ready , we need to deploy it via netfly, linktree or via custom domain

@Adarsh-jaiss Please see my recent edits on this message

**Reply by Adarsh-jaiss** [reply]
👍

**Comment by Adarsh-jaiss** [comment]
-- so, now we have deployed our website through netlify in the "website" branch of this repo
When we're done with the website and merge the "website" branch into main, we need to change deploy settings of Netlify

To change settings: @Adarsh-jaiss 
To help @Adarsh-jaiss change settings: @KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
Please read my edits in this.

**Reply by Adarsh-jaiss** [reply]
now what next we are going to do ?

**Reply by KendallDoesCoding** [reply]
Site hosted at: https://unruffled-chandrasekhar-ffade4.netlify.app/

**Reply by KendallDoesCoding** [reply]
> lets's see what we have to do next
-- We have to format the README a little better https://github.com/Adarsh-jaiss/journey-book/issues/2 (Add badges, grammer check, remove extra sections, etc)
-- you are going to work on HTML and CSS
-- I have approached riti for javascript

**Reply by Adarsh-jaiss** [reply]
-- so we are going to enhance readme 
-- and you are going to work on HTML and css

**Reply by KendallDoesCoding** [reply]
> -- so we are going to enhance readme
Already made a PR on this

> -- and you are going to work on HTML and css
Yep, I would prefer we get a JS contact first, either Riti, TechStudent or someone else and we can discuss then I will start work.

**Reply by Adarsh-jaiss** [reply]
am i right !

**Reply by KendallDoesCoding** [reply]
> am i right !

yep.
